apps/gitness/src/utils/__tests__/common-utils.test.ts

```diff
@@ -0,0 +1,151 @@
+import { describe, expect, it } from 'vitest'
+
+import { SummaryItemType, type RepoFile } from '@harnessio/ui/views'
+
+import { getLogsText, sortFilesByType } from '../common-utils'
+
+// Mock data for testing
+const mockLogs = [{ out: 'Log line 1\n' }, { out: 'Log line 2\n' }, { out: 'Log line 3\n' }]
+
+const mockFiles: RepoFile[] = [
+  {
+    name: 'file1.txt',
+    type: SummaryItemType.File,
+    id: '1',
+    path: 'file1.txt',
+    lastCommitMessage: 'file1 commit',
+    timestamp: '2021-09-01T00:00:00Z'
+  },
+  {
+    name: 'folder1',
+    type: SummaryItemType.Folder,
+    id: '2',
+    path: 'folder1',
+    lastCommitMessage: 'folder1 commit',
+    timestamp: '2021-09-01T00:00:00Z'
+  },
+  {
+    name: 'file2.txt',
+    type: SummaryItemType.File,
+    id: '3',
+    path: 'file2.txt',
+    lastCommitMessage: 'file2 commit',
+    timestamp: '2021-09-01T00:00:00Z'
+  },
+  {
+    name: 'folder2',
+    type: SummaryItemType.Folder,
+    id: '4',
+    path: 'folder2',
+    lastCommitMessage: 'folder2 commit',
+    timestamp: '2021-09-01T00:00:00Z'
+  }
+]
+
+describe('getLogsText', () => {
+  Object.defineProperty(document, 'queryCommandSupported', {
+    value: vi.fn().mockReturnValue(true),
+    writable: true // Ensure it can be reassigned
+  })
```

Comment:
Looks like this was setup in the `vitest-setup.ts` file

Reply:
Done, cleaned up from here.